### PR TITLE
Add Packet Delay Variation measurement

### PR DESCRIPTION
**Pull request by FR4NK-W**
Aggregate measurements on host, makes response size independent of
query parameters

### CODE REVIEW

File: bwtester/bwtestclient/bwtestclient.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func printUsage() {
 		"all but the last one are set to the default values, e.g. ?,1000,?,5Mbps will run the test for the " +
 		"default duration and send as many packets as required to reach a bandwidth of 5 Mbps with the given " +
 		"packet size.")
-	fmt.Println("\tSupported unit prefixes for the bandwidth parameter are none (e.g. 1500bps for 1.5kbps), k, M, G, T.")
+	fmt.Println("\tSupported bandwidth unit prefixes are none (e.g. 1500bps for 1.5kbps), k, M, G, T.")
```
Comment (perrig) [inline bwtester/bwtestclient/bwtestclient.go]: are: none (add colon)

File: bwtester/bwtestclient/bwtestclient.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func printUsage() {
...
 	fmt.Println("\tYou can also only set the target bandwidth, e.g. -cs 1Mpbs")
```
Comment (perrig) [inline bwtester/bwtestclient/bwtestclient.go]: Mpbs -> Mbps

File: bwtester/bwtestclient/bwtestclient.go (hunk 1 above)
Comment (FR4NK-W) [reply bwtester/bwtestclient/bwtestclient.go]: Done

File: bwtester/bwtestclient/bwtestclient.go (hunk 2 above)
Comment (FR4NK-W) [reply bwtester/bwtestclient/bwtestclient.go]: Done

**Pull request merged**